andela/phlox-ah-frontend
# #160451467 Users should be able to follow each other

What does this PR do?
This PR function to follow users and lists the followers of the user

Description of Task to be completed?
show follow button on an article
#### How should this be manually tested?
Go to the view an article url, you should see a follow button
On clicking the button, it should follow the user and it should change to unfollow
#### Any background context you want to provide?
None
